pysrc/models/column_statistics.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+
+@dataclass_json
+@dataclass(frozen=True)
+class ColumnStatistic:
```

Comment:
I think it would be a good idea to store the `ColumnInformation` in the `ColumnStatistics`. Maybe there exists a matching between the indices, but I think it'd be much clearer when we embed the information directly.

Reply:
Done!